src/ert/run_models/multiple_data_assimilation.py

```diff
@@ -191,7 +189,7 @@ def run_experiment(
 
         self.setPhase(iteration_count + 1, "Experiment completed.")
 
-        return prior_context
+        return posterior_context
```

Comment:
Is this related to the other changes? Perhaps split it to another commit?

Reply:
Not related no, I've removed it.